controller/k8s/api.go

```diff
@@ -198,6 +225,8 @@ func (api *API) GetObjects(namespace, restype, name string) ([]runtime.Object, e
 		return api.getRCs(namespace, name)
 	case k8s.Service:
 		return api.getServices(namespace, name)
+	case k8s.ServiceProfile:
+		return api.getServiceProfiles(namespace, name)
```

Comment:
I don't think we need to add this to `GetObjects`, since `GetObjects` is only used right now for fetching resource targets used in CLI stat and tap commands. I'd remove this case, and remove the `getServiceProfiles` func that you're adding.